include/rocksdb/utilities/transaction.h

```diff
@@ -443,6 +443,13 @@ class Transaction {
     }
   }
 
+  virtual Status GetEntityForUpdate(const ReadOptions& read_options,
+                                    ColumnFamilyHandle* column_family,
+                                    const Slice& key,
+                                    PinnableWideColumns* columns,
+                                    bool exclusive = true,
+                                    bool do_validate = true) = 0;
```

Comment:
nit: I see other `GetForUpdate` APIs have `const bool do_validate`.

Reply:
Since `do_validate` is passed by value, declaring it as `const bool` vs `bool` doesn't really make a difference (declaring it `const` prevents the function's *copy* of `do_validate` from being changed in the function body but we don't do that anyway, so to me `const`s like this just adds noise).